pkgs/development/libraries/gstreamer/rs/default.nix

```diff
@@ -137,6 +138,12 @@ stdenv.mkDerivation rec {
     '';
   };
 
+  # gstreamer-gl-* detection is broken due to an upstream meson issue
+  postPatch = lib.optionalString (!gst-plugins-base.glEnabled) ''
+    substituteInPlace meson.build \
+      --replace "['gstreamer-gl-1.0', 'gst-plugins-base', 'gst_gl_dep', 'gstgl']," ""
+  '';
```

Comment:
Look at the `preConfigure` hook which already does this on non-aarch64-darwin. It needs to be conditionalized on `gst-plugins-base.glEnabled` instead and then this should be automatic

Reply:
I can't find the upstream issue that mentioned that anymore